### PR TITLE
add TS examples for equivalent SDK methods

### PR DESCRIPTION
**Pull request by piyalbasu**

### CODE REVIEW

File: docs/building-apps/wallet/sep24.mdx [hunk 1]
```diff
@@ -143,6 +189,18 @@ suspend fun depositDifferentAccount(): InteractiveFlowResponse {
...
 ```
 
+```typescript
+const recipientAccount = "G...";
+const depositDifferentAccount = async (): Promise<InteractiveFlowResponse> => {
+  return anchor.interactive().deposit({
+    accountAddress: accountKp.publicKey(),
+    assetCode,
+    authToken,
+    fundsAccountAddress: recipientAccount
```
Comment (piyalbasu) [inline docs/building-apps/wallet/sep24.mdx:199]: I believe `fundsAccountAddress` is the equivalent key for `destinationAccount` above, but lmk if I'm wrong

Comment (Ifropc) [reply docs/building-apps/wallet/sep24.mdx]: I think so. It should be optional, is it like this right now? (If not, making it optional is part of WAL-889 IIRC)

File: docs/building-apps/wallet/intro.mdx [hunk 2]
```diff
@@ -60,6 +66,19 @@ val walletCustom = Wallet(
 )
 ```
 
+```typescript
+const customClient: AxiosInstance = axios.create({
+  baseURL: "https://some-url.com/api",
```
Comment (Ifropc) [inline docs/building-apps/wallet/intro.mdx:71]: Why do we even need to set baseUrl? I don't think it's good to show it in the example

File: docs/building-apps/wallet/intro.mdx [hunk 3]
```diff
@@ -60,6 +66,19 @@ val walletCustom = Wallet(
...
+  timeout: 1000,
+  headers: { "X-Custom-Header": "foobar" },
+});
+let appConfig = new ApplicationConfiguration(DefaultSigner, customClient);
```
Comment (Ifropc) [inline docs/building-apps/wallet/intro.mdx:75]: Shouldn't DefaultSigner be there by default? (I.e. you don't need to set it manually)

File: docs/building-apps/wallet/intro.mdx [hunk 4]
```diff
@@ -60,6 +66,19 @@ val walletCustom = Wallet(
 )
 ```
 
+```typescript
```
Comment (Ifropc) [inline docs/building-apps/wallet/intro.mdx:69]: I think all this code belongs to the "Configuring the Client" section

File: docs/building-apps/wallet/sep24.mdx [hunk 5]
```diff
@@ -28,6 +28,10 @@ Let's start with getting an instance of `Interactive` class, responsible for all
 val sep24 = anchor.sep24()
 ```
 
+```typescript
+let sep24 = await anchor.getInfo();
```
Comment (Ifropc) [inline docs/building-apps/wallet/sep24.mdx:32]: `anchor.interactive()` ?

File: docs/building-apps/wallet/sep24.mdx (hunk 1 above)
Comment (piyalbasu) [reply docs/building-apps/wallet/sep24.mdx]: `fundsAccountAddress` is optional. It gets set to `accountAddress` if it isn't explicitly set and then sent to the `/interactive` endpoint as `account`

File: docs/building-apps/wallet/sep24.mdx [hunk 6]
```diff
@@ -143,6 +189,18 @@ suspend fun depositDifferentAccount(): InteractiveFlowResponse {
 }
 ```
 
+```typescript
+const recipientAccount = "G...";
```
Comment (Ifropc) [inline docs/building-apps/wallet/sep24.mdx:193]: Can you add TODO here to modify code when memo is supported?

File: docs/building-apps/wallet/sep24.mdx [hunk 7]
```diff
@@ -173,6 +241,19 @@ val watcher = sep24.watcher()
 val result = watcher.watchOneTransaction(token, "transaction id")
 ```
 
+```typescript
+let watcher = anchor.watcher();
+
+let result = watcher.watchOneTransaction({
+  authToken,
+  assetCode,
```
Comment (Ifropc) [inline docs/building-apps/wallet/sep24.mdx:249]: Why do we need assetCode when watching one transaction?

File: docs/building-apps/wallet/sep24.mdx [hunk 8]
```diff
@@ -236,6 +324,13 @@ It's also possible to fetch transaction by the asset
 val transactions = sep24.getTransactionsForAsset(asset, token)
 ```
 
+```typescript
+const transactions = await anchor.getHistory({
```
Comment (Ifropc) [inline docs/building-apps/wallet/sep24.mdx:328]: `getHistory` should be deprecated... It has some extra filtering logic that shouldn't be there.
`getTransactionsForAsset` is more flexible than `getHistory`

**Review by Ifropc — APPROVED** [review]

File: docs/building-apps/wallet/sep24.mdx (hunk 1 above)
Comment (Ifropc) [reply docs/building-apps/wallet/sep24.mdx]: Got it. Small thing: for withdrawal it should be `originAccount`

File: docs/building-apps/wallet/intro.mdx (hunk 2 above)
Comment (piyalbasu) [reply docs/building-apps/wallet/intro.mdx:71]: You don't necessarily, it's just an example of the params `axios.create` takes. I can rm

File: docs/building-apps/wallet/intro.mdx (hunk 3 above)
Comment (piyalbasu) [reply docs/building-apps/wallet/intro.mdx:75]: You don't, I can just pass `null` here

Comment (Ifropc) [reply docs/building-apps/wallet/intro.mdx:75]: Can you just omit it or it's not possible?

File: docs/building-apps/wallet/sep24.mdx (hunk 7 above)
Comment (Ifropc) [reply docs/building-apps/wallet/sep24.mdx:249]: Ok figure it out later, as I don't think it should be necessary

Comment (acharb) [reply docs/building-apps/wallet/sep24.mdx:250]: @CassioMG is more familiar with this code than I am. But from what I understand we're using the asset code as the key for the registry to keep track of transactions: https://github.com/stellar/typescript-wallet-sdk/blob/b0a1ff12c2531c9526ebc59bfbed285f1db04d5d/src/walletSdk/Watcher/index.ts#L100-L113

**Pull request merged**